### PR TITLE
Updating "Navigating the viewer"

### PR DESCRIPTION
**Pull request by gavrelina**
Updating screenshots and editing a few paragraphs for Navigating the Viewer page.

- closes https://github.com/orgs/rerun-io/projects/1/views/11?pane=issue&itemId=81853628&issue=rerun-io%7Crerun%7C7564

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7757?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7757?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7757)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: docs/content/getting-started/navigating-the-viewer.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ title: Navigating the viewer
...
+-   [Navigating the timeline](#navigating-the-timeline)
 
 Here is a preview of the dataset that we will be working with:
 
 <picture>
-  <source media="(max-width: 480px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/480w.png">
-  <source media="(max-width: 768px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/768w.png">
-  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/1024w.png">
-  <img src="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/full.png" alt="viewer walkthrough dataset preview screenshot">
+  <img src="https://static.rerun.io/viewer_walkthrough_preview/eba55ea05deef665393cf42cc48d1236b4cc9692/full.png" alt="">
```
Comment (Wumpf) [inline docs/content/getting-started/navigating-the-viewer.md:16]: as per note on slack dm: We really shouldn't have the `debug viewer` show on those screenshots. Either run a release build or use the `take a screenshot` feature from Rerun's command pallete for a "standardized" screenshot of the viewer which suppresses this banner

**Review by Wumpf — APPROVED** [review]
nice, a lot better!
added some suggestions. Screenshots need updating again but looks good to me otherwise!

File: docs/content/getting-started/navigating-the-viewer.md (hunk 1 above)
Comment (gavrelina) [reply docs/content/getting-started/navigating-the-viewer.md:16]: will replace them!

File: docs/content/getting-started/navigating-the-viewer.md [hunk 2]
```diff
@@ -54,91 +54,83 @@ like._
...
+-   Both the images and pinhole projection are hierarchically dependent on the camera's position, which is described by the `/camera` entity. This entity includes a series of transforms that together form a [Transform3D archetype](../reference/types/archetypes/transform3d.md).
 
-## Exploring data
+The hierarchy of logged entity streams and their component streams is found under `Streams` in the Timeline panel. A similar list appears in the `Blueprint` panel, but the key difference is that the Blueprint panel focuses on how data is arranged and visualized in the Viewport, while the Streams panel shows when and what events were logged. In other words, an entity may be logged once but displayed in multiple views.
 
-The space views are where you can see the data that was actually logged. This scene has streams of data for 6 different
-primitives, also known as [entities](../concepts/entity-component.md):
+Visualizations can also be customized per view using [Overrides](../concepts/visualizers-and-overrides.md) in the Selection panel. In the screenshot below, the same entity `keypoints` is shown in different colors.
```
[the remark this answers is not in the document]
Comment (gavrelina) [reply docs/content/getting-started/navigating-the-viewer.md:113]: >to point this out briefly here 
by "this" you mean to add a note on how the different colors are archived? or something else?

File: docs/content/getting-started/navigating-the-viewer.md [hunk 3]
```diff
@@ -176,24 +155,16 @@ Try each of the following:
...
-
-<picture>
-  <source media="(max-width: 480px)" srcset="https://static.rerun.io/viewer_walkthrough6_timeline/9816d7becf19399735bef1f17f1d4bb928c278f7/480w.png">
-  <source media="(max-width: 768px)" srcset="https://static.rerun.io/viewer_walkthrough6_timeline/9816d7becf19399735bef1f17f1d4bb928c278f7/768w.png">
-  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/viewer_walkthrough6_timeline/9816d7becf19399735bef1f17f1d4bb928c278f7/1024w.png">
-  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/viewer_walkthrough6_timeline/9816d7becf19399735bef1f17f1d4bb928c278f7/1200w.png">
-  <img src="https://static.rerun.io/viewer_walkthrough6_timeline/9816d7becf19399735bef1f17f1d4bb928c278f7/full.png" alt="viewer walkthrough timeline screenshot">
-</picture>
+-   Move the time indicator by dragging it to a different point on the timeline.
+    For certain timelines, you can also click on the frame number and manually type the desired frame.
```
[the remark this answers is not in the document]
Comment (gavrelina) [reply docs/content/getting-started/navigating-the-viewer.md:165]: got it! will leave it out then

File: docs/content/getting-started/navigating-the-viewer.md [hunk 4]
```diff
@@ -244,7 +215,7 @@ Again, if you ran into any issues following this guide, please don't hesitate to
 
 ### Up next
 
-- [Get started](./quick-start) by writing a program to log data with the Rerun SDK.
-- Learn how to further [configure the viewer](./configure-the-viewer) to suit your data.
-- Explore other [examples of using Rerun](/examples).
-- Consult the [concept overview](../concepts.md) for more context on the ideas covered here.
+-   [Get started](./quick-start) by writing a program to log data with the Rerun SDK.
```
[the remark this answers is not in the document]
Comment (gavrelina) [reply docs/content/getting-started/navigating-the-viewer.md:218]: yeah, not sure how does this happen, also that part I haven't even changed from the original version! 🤷‍♀️

**Review by Wumpf — APPROVED** [review]
👍

**Pull request merged**